packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -686,6 +686,102 @@ inline void fromRawValue(
   react_native_expect(false);
 }
 
+// [macOS [visionOS]
+inline void fromRawValue(
+    const PropsParserContext& context,
+    const RawValue& value,
+    Cursor& result) {
+  result = Cursor::Auto;
+  react_native_expect(value.hasType<std::string>());
+  if (!value.hasType<std::string>()) {
+    return;
+  }
+  auto stringValue = (std::string)value;
+  if (stringValue == "alias") {
+    result = Cursor::Alias;
+    return;
+  }
+  if (stringValue == "auto") {
+    result = Cursor::Auto;
+    return;
+  }
+    if (stringValue == "col-resize") {
```

Comment:
Nit: The indentation of all the following if-statements are off.

Reply:
Good catch. Probably only specific to the RNM versions of this change since we have so many more cursors. I'll update.